tests/visco_plastic_differentplasticitydifferentphases.prm

```diff
@@ -0,0 +1,105 @@
+# Copy of visco_plastic.prm, but include a phase transition for the EoS
+
+set Dimension                              = 2
+set Start time                             = 0
+set End time                               = 0
+set Use years in output instead of seconds = true
+set Nonlinear solver scheme                = single Advection, iterated Stokes
+set Max nonlinear iterations               = 1
```

Comment:
Several nonlinear iterations would be better. Maybe set to 10? Or is that really slow?